server/server/models/user.py

```diff
@@ -69,3 +76,97 @@ def is_active(self, request):
                                 .filter_by(status="Active").one().id
                 return True
         return False
+
+    def __init__(self, email, nickname, password, create_date, location,
+                 first_name, last_name, status_id, role_id, avatar,
+                 banned_to_date):
+        self.email = email
+        self.nickname = nickname
+        self.password = password
+        self.create_date = create_date
+        self.location = location
+        self.first_name = first_name
+        self.last_name = last_name
+        self.status_id = status_id
+        self.role_id = role_id
+        self.avatar = avatar
+        self.banned_to_date = banned_to_date
+
+
+def from_json(cls, data):
+    """ Deserialize User object from JSON """
+    return cls(**data)
+
+
+def model_to_json(sqlalchemy_object):
+    fields_arr = [prop.key for prop in
+                  class_mapper(sqlalchemy_object.__class__).iterate_properties
+                  if isinstance(prop, ColumnProperty)]
+    _dict = {}
+    for key in fields_arr:
+        _dict[key] = str(getattr(sqlalchemy_object, key))
```

Comment:
I don't like idea to convert **all** values to str, as JSON distinguish string, float, bool and you will need to parse it in frontend side, so types might be useful. Please convert only datetime values.